Elf/__init__.py

```diff
@@ -1109,10 +946,9 @@ def elfFromMemoryObject(memobj, baseaddr):
     fd = vstruct.MemObjFile(memobj, baseaddr)
     return Elf(fd)
 
-
+# TODO: Consider deprecating these. We have methods on the ElfReloc objects specifically for these two
```

Comment:
that's fine.

Reply:
Aye. It'd be a thing for later. I don't want to just rip out the APIs, but deduplication is nice.